### PR TITLE
REVIEW: The XMPP Newsletter September 2024

### PR DESCRIPTION
**Pull request by nemmig**
# XMPP Newsletter

## This is the draft for the next release covering September 2024

*Final release is planned for 5th October 2024.*

Contributions are much appreciated. You don't need any technical knowledge to contribute. If you need help, please ask in the Communication-Team group chat: [commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org).

If you would like to add news, that's possible via:

- Using our [online Pad](https://pad.nixnet.services/oHnY_ZvLT8SoFyCqIC2ung) **(Pad is closed for September 2024!)**
- Adding your content to this pull request's thread (i.e. a comment below)
- Creating a pull request
- Posting something in our XMPP channel (xmpp:[commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)?join)

It's really helpful if you try to give a short summary (<5 lines) of your contribution or of what the link you're posting is offering. You don't need to write in a formal tone - just be passionate about why this motivates you or about what you are working on!

| During the month | 1st - 2nd of the month | 3rd - 4th of the month | 5th of the month |
|--------------|-----------|------------|------------|
| Create Draft in Pad & GitHub | Last Call for final edits     | Review phase        |RELEASE! :rocket:|
| Collect news & highlights      |  |  Translators can start working      | |
| Reminder mid-month + end of month      |  |        ||

## Status

- [x] Collection of news
- [x] Review
- [ ] Blog post + Newsletter mail (use `tools/newsletter_email.py` after the post has been published)
- [ ] Social media
- [ ] Translations

### CODE REVIEW

File: content/blog/newsletter/2024-10-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+### XMPP Libraries & Tools
+
+- [Hazelcast plugin for Openfire](https://www.igniterealtime.org/projects/openfire/plugin-archive.jsp?plugin=hazelcast) version [3.0.0](https://www.igniterealtime.org/projects/openfire/plugin-archive.jsp?plugin=hazelcast) has been released, adding clustering support to [Openfire](https://www.igniterealtime.org/projects/openfire/). It is based on [the Hazelcast platform](https://hazelcast.com/). This release brings a major upgrade of the platform. It migrates from version 3.12.5 to 5.3.7. Please [read the upgrade instructions carefully](https://www.igniterealtime.org/projects/openfire/plugins/3.0.0/hazelcast/readme.html) for this one.
```
Comment (Echolon) [inline content/blog/newsletter/2024-10-05-newsletter.md:72]: Usually we collect the Openfire news as part of the ignite realtime community

**Comment by Echolon** [comment]
@nemmig again, really awesome preparations!!

File: content/blog/newsletter/2024-10-05-newsletter.md (hunk 1 above)
Comment (nemmig) [reply content/blog/newsletter/2024-10-05-newsletter.md:75]: Oh, I'm sorry. I didn't know that. This was my first time publishing a news about Openfire. I'll have to ask you what's the right way to do it for the next newsletter or we could fix it now if you want to :)

**Comment by Echolon** [comment]
## Editorial closing of the XMPP Newsletter September  '24 release

Many thanks to all supporters in this story!

- English (original): [xmpp.org](https://xmpp.org/categories/newsletter/)
  - General contributors: *Adrien Bourmault (neox), Alexander "PapaTutuWawa", Arne, cal0pteryx, emus, Federico, Gonzalo Raúl Nemmi, Jonas Stein, Kris "poVoq", Licaon_Kter, Ludovic Bocquet, Mario Sabatino, melvo, MSavoritias (fae,ve), nicola, Schimon Zachary, Simone Canaletti, singpolyma, XSF iTeam*
- French: [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
  - Translators: *Adrien Bourmault (neox), alkino, anubis, Arkem, Benoît Sibaud, mathieui, nyco, Pierre Jarillon, Ppjet6, Ysabeau*
- Italian: [notes.nicfab.eu](https://notes.nicfab.eu)
  - Translators: *nicola*
- Spanish: [xmpp.org](https://xmpp.org/categories/newsletter/])
  - Translators: *Gonzalo Raúl Nemmi*
- German: [xmpp.org](https://xmpp.org/categories/newsletter/])
  - Translators: *Millesimus*

❤️ 🎆 Next and last release for this year will be next month on 5th of November 2024!

Have a good time until then and see you soon!

**Pull request merged**